### PR TITLE
Update to Admin UI 1.24.8

### PR DESCRIPTION
**Pull request by amotl**
## About

This patch accompanies the corresponding release of Admin UI. It is just a maintenance release, bundling updates to both runtime and build-time dependencies.

## References
- https://github.com/crate/crate-admin/pull/860
- https://github.com/crate/crate-admin/pull/862
- https://github.com/crate/crate-admin/pull/864

/cc @surister, @kneth, @msbt

### CODE REVIEW

File: docs/appendices/release-notes/5.10.0.rst [hunk 1]
```diff
@@ -105,3 +105,9 @@ Administration and Operations
 
 - Added node settings for the :ref:`jwt_defaults` allowing to provide global
   values for the :ref:`JWT properties <create-user-jwt>`.
+
+User Interface
+--------------
+
+- Updated to Admin UI 1.24.8, which is a maintenance build using an updated
+  set of dependencies.
```
Comment (seut) [inline docs/appendices/release-notes/5.10.0.rst:113]: Seems like this update has no user visible change, such I'd leave an entry out.
```suggestion
```

**Review by seut — APPROVED** [review]
lgtm otherwise

File: docs/appendices/release-notes/5.10.0.rst (hunk 1 above)
Comment (amotl) [reply docs/appendices/release-notes/5.10.0.rst]: I've just removed that change log item. Thanks for the guidance.

**Comment by amotl** [comment]
@crate/core-team and @crate/customer-engineering: Please have another round of additonal user testing after this has converged into a nightly release tomorrow. @msbt validated the build on behalf of https://github.com/crate/crate-admin/pull/864, but I would like to have additional eyeballs on it. Thanks!

```
docker pull crate/crate:nightly
docker run --rm -it --name=cratedb --publish=4200:4200 --publish=5432:5432 --env=CRATE_HEAP_SIZE=2g crate/crate:nightly -Cdiscovery.type=single-node
```
```
open http://localhost:4200/
```

/cc @kneth

**Comment by proddata** [comment]
This release broke the Admin UI

worked fine: `crate/crate:nightly-5.10.0-2024-10-28-00-02`
broken: `crate/crate:nightly-5.10.0-2024-10-29-00-02`

also see
https://github.com/crate/crate-admin/issues/878

**Pull request merged**